lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -888,3 +900,45 @@ class _EditableSizeAndTransform {
   final double height;
   final Float64List transform;
 }
+
+/// Used for detecting the changes in the selection.
```

Comment:
nit: shorter: "Detects changes in text selection"

Reply:
done.